tsdb/wlog/watcher_test.go

```diff
@@ -58,29 +58,47 @@ type writeToMock struct {
 	floatHistogramsAppended int
 	seriesLock              sync.Mutex
 	seriesSegmentIndexes    map[chunks.HeadSeriesRef]int
+
+	// delay reads with a short sleep
+	delay time.Duration
 }
 
 func (wtm *writeToMock) Append(s []record.RefSample) bool {
+	if wtm.delay > 0 {
+		time.Sleep(10 * time.Millisecond)
```

Comment:
Did we mean to sleep for wtm.delay here?

Reply:
Yes that was what I meant.

Reply:
:facepalm: sorry about that, fixed